build-scripts/docker/repo-test/generate-repo-files.py

```diff
@@ -172,6 +172,12 @@ def write_release_files (release):
         write_dockerfile('ubuntu', 'jammy', release)
         write_list_file('ubuntu', 'jammy', release)
 
+    if release in ['auth-master',
+                   'rec-master',
+                   'dnsdist-master']:
+        write_pkg_pin_file(release)
```

Comment:
only Debian/Ubuntu need the pin file, which, now that I look at it, also means it's missing from some other targets, and depending on build order, a dnsdist Dockerfile might get a pdns pin file.

So, there's a bug there, but for this PR, the line can just go :-)

(Usually, the repo package version will be higher than what's in the distro as stock, and then the pin file doesn't matter, which probably explains that we never noticed.)

Reply:
I was just cargo culting. Moved it to the proper sections now.